Azure sentinel error handler bug (#17323)
demisto/content: Packs/AzureSentinel/ReleaseNotes/1_3_1.md

#### Integrations
##### Azure Sentinel

> - Fixed an issue that when trying to decode a 'None' type response content in error_handler, then a JSONDecodeError is raised.

Reviewer: ```suggestion
- Fixed an issue where the integration failed with 'JSONDecodeError' in cases of a 'None' type response.
```
